### DISCUSSION TITLE
Self hosted sentry dsn is with http instead of https.

### DISCUSSION THREAD

**Discussion by martirosyanH** [getsentry/support#63] [Support] [answered]
Hi everyone,

I run self-hosted sentry, my website is up with https, but once I try to create dsn key, it is by http.
I simply changed it to https, and it works for the client,
but now I try to upload debug symbols(`pdb` files), it fails with 
`
API request failed
  caused by: [7] Couldn't connect to server (Failed to connect ${myIP} port 80)
`
Why it tries to request to http?

Thanks in advance

**Comment by chadwhitacre** [comment, marked as answer]
Please ask in https://github.com/getsentry/self-hosted/issues, thanks.
